src/client/entity-editor/button-bar/button-bar.js

```diff
@@ -56,43 +55,30 @@ import {connect} from 'react-redux';
  */
 function ButtonBar({
 	aliasesInvalid,
-	disambiguationVisible,
```

Comment:
There is a leftover `disambiguationVisible` in the JSDOC documentation comments on lines 41-43